docker/app/Dockerfile

```diff
@@ -35,11 +40,26 @@ RUN set -xe \
     && apk add --no-cache --virtual .build-deps ${RUNTIME_DEPS} \
     # workaround for rabbitmq linking issue
     && ln -s /usr/lib /usr/local/lib64 \
+    # Build openssl
+    && mkdir -p /usr/local/src && cd $_ \
+        && wget "https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz" -O "openssl-${OPENSSL_VERSION}.tar.gz" \
+        && echo "$OPENSSL_SHA256" "openssl-${OPENSSL_VERSION}.tar.gz" | sha256sum -c - \
+        && tar -zxvf "openssl-${OPENSSL_VERSION}.tar.gz" \
+        && cd "openssl-${OPENSSL_VERSION}" \
+        && ./config no-async shared --prefix=/usr/local/ssl --openssldir=/usr/local/ssl -Wl,-rpath,/usr/local/ssl/lib \
+        && make && make install \
+        && cp /usr/local/ssl/bin/openssl /usr/bin/openssl \
+        && rm -rf "/usr/local/src/openssl-${OPENSSL_VERSION}.tar.gz" "/usr/local/src/openssl-${OPENSSL_VERSION}"
+
+#RUN ln -s /usr/local/ssl/lib/libcrypto.so.1.1 /usr/lib/libcrypto.so.1.1 \
+#        && ln -s /usr/local/ssl/lib/libssl.so.1.1 /usr/lib/libssl.so.1.1
+
     # this c-library is required for 'php-amqp'
```

Comment:
Single `RUN` instruction broken

Reply:
fixed.
why is Single `RUN` necessary?
it doesn't allow use of the cache

Reply:
Single `RUN` instruction creates **one** layer. Best practice - if you install any dependencies at the start and remove it them in a one `RUN` instruction